libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/FocalPoint.kt

```diff
@@ -0,0 +1,6 @@
+package com.mapbox.navigation.ui.maps.camera.data
+
+data class FocalPoint(
```

Comment:
How about nesting this type in `FollowingFrameOptions`, `FollowingFrameOptions.FocalPoint`? `FocalPoint` is rather generic and has a pretty narrow application on our camera APIs.

Reply:
Sounds good to me.